chain/txValidator.go

```diff
@@ -161,6 +161,7 @@ func CheckTransactionPayload(txn *transaction.Transaction, height uint32) error
 		if !match {
 			fmt.Errorf("name should match regex %s", regexPattern)
 		}
+	case pb.TRANSFER_NAME_TYPE:
```

Comment:
We should validate `registrant` field of register, transfer, delete name (check length)